addons/udes_stock/models/stock_picking.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, _
+from odoo.exceptions import ValidationError
+
+
+class StockPicking(models.Model):
+    _inherit = 'stock.picking'
+
+    # TODO:
+    # - [ ] Requires backorder ? (revise and consolidate with odoo core function) Why not just use the core cehck_backorders?
+    # - [ ] Backorder moves/move lines? (Done->just move lines)
+
+    sequence = fields.Integer('Sequence', default=0)
```

Comment:
More descriptive name/string, please!

Reply:
Agree. A short explanation adding the intention of the file.